modules/project-common.nix

```diff
@@ -24,6 +24,18 @@ with lib.types;
           shell = { tools.cabal = {}; crossPlatforms = p: [ p.ghcjs ]; }
       '';
     };
+    evalSystem = mkOption {
+      type = nullOr str;
+      default = null;
+      description = ''
+        Specifies the system on which `cabal configure` and `plan-to-nix` should run.
+        If not specified the `evalPackages` default will be used (builtins.currentSystem or
```

Comment:
`evalPackages` is our internal jargon, can we say something more comprehensible to a user?

Reply:
I'm not too keen on it either, but I can't think of a better name (only other name I could think of was `bootPackages`).